### ISSUE TITLE
seek bar for background mode

### ISSUE THREAD

**Issue by MD77MD** [TeamNewPipe/NewPipe#4156]
#### Describe the feature you want

![Screenshot_20200818-035015](https://user-images.githubusercontent.com/63132171/90458153-44e2f780-e106-11ea-934c-fe9a10ebb656.png)
as you can see the background mode already have progress bar but you cannot use it to change playback position.
#### Is your feature request related to a problem? Please describe it

there is no seek bar in background mode. the only way is to open notification, then open play queue, then you can find seek bar to change position.
#### Additional context

it just seems natural to have a sick bar there
#### How will you/everyone benefit from this feature?

make it alot easier for users who use background mode like music and audiobooks

**Comment by opusforlife2** [comment]
Read the comments in #3178. 

TL;DR: Only Android 10+ can show the seekbar.

**Comment by avently** [comment]
@opusforlife2 
> TL;DR: Only Android 10+ can show the seekbar.

He means that this red progress bar on screenshot should be touchable like in YouTube app. Ths enhancement can be made by someone I think. Would be good to have this progress bar working like he propose.

**Comment by MD77MD** [comment]
@opusforlife2 
you miss understood

it is exactly what @avently said above https://github.com/TeamNewPipe/NewPipe/issues/4156#issuecomment-675636769

**Comment by opusforlife2** [comment]
No, no it was my fault. I didn't unblock Github's image domain when checking or I would have seen the attached image. Thanks for clarifying, guys!
